cli/lint.rs

```diff
@@ -133,6 +138,52 @@ fn lint_file(file_path: PathBuf) -> Result<Vec<LintDiagnostic>, ErrBox> {
   Ok(file_diagnostics)
 }
 
+/// Lint stdin and write result to stdout.
+/// Treats input as TypeScript.
+/// Compatible with `--json` flag.
+fn lint_stdin(json: bool) -> Result<(), ErrBox> {
+  let mut source = String::new();
+  if stdin().read_to_string(&mut source).is_err() {
+    return Err(ErrBox::error("Failed to read from stdin"));
+  }
+
+  let reporter_kind = if json {
+    LintReporterKind::Json
+  } else {
+    LintReporterKind::Pretty
+  };
+  let mut reporter = create_reporter(reporter_kind);
+  let lint_rules = rules::get_recommended_rules();
+  // TODO(magurotuna): it'd be better to be able to deal with JavaScript
+  let syntax = swc_util::get_syntax_for_media_type(msg::MediaType::TypeScript);
```

Comment:
remove this TODO, it doesn't really matter if all input is treated as TypeScript - SWC should handle that

Reply:
> it doesn't really matter if all input is treated as TypeScript - SWC should handle that
I didn't know that. Great.
I'll remove it.

Reply:
I've removed it.